pennylane/__init__.py

```diff
@@ -50,6 +46,7 @@
 from .utils import inv
 from ._version import __version__
 from .io import *
+from ._grad import jacobian, grad
```

Comment:
Very very glad that `jacobian` and `grad` have been moved to a separate module. Keeping them in the `__init__.py` was causing the file to get quite unruly. And, it solves various circular import problems.